Source/app.js

```diff
@@ -16,9 +17,23 @@ const routes = require('./routes');
 mongoose.connect(`mongodb://${config.MongoDB.User}:${config.MongoDB.Pass}@${config.MongoDB.Host}:${config.MongoDB.Port}/${config.MongoDB.Name}`);
 mongoose.Promise = Promise;
 
+/**
```

Comment:
Dit soort comments geeft ongewenst resultaat in de documentatie, geneer de documentatie voor het effect:
`npm run generate`

(Dit geldt voor alle comments, kijk ook op JSDoc welke tags erbij moeten/kunnen)